flocker/node/functional/blank-docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM scratch
+MAINTAINER Jérôme Petazzoni <jerome@docker.com>
+ADD rootfs.tar /
```

Comment:
Mmm I'm not excited about this.  Does the container actually need a full copy of the busybox filesystem?  Also, not a fan of binary artifacts checked in to version control.  Whatever we need to build, we should build from source.  If there's non-trivial build overhead, we can think about amortizing it by having Docker hub build it for us automatically.

The only behavior we need from this container is for it to be runnable.  You can run a container from an empty image - though it will fail (because you must run a command and it has no command).  I'm not sure if that failure would be a problem here, maybe not.  Worth a try.  Otherwise 

(Also it's not clear what the license of rootfs.tar is)